.github/workflows/daily-status.yaml

```
@@ -22,8 +29,14 @@ jobs:
       - name: Generate e-mail content
         id: content
         run: |
-          subject="$(python -m pip-run -q -- .github/workflows/tools/daily-status.py)"
+          fpath="$(date +%Y/%m/%d.html)"
```

```suggestion
          mkdir -p "$(date +%Y/%m)"
```
or alike since I didn't mention anything creating them?

otherwise looks LGTM to me -- THANK YOU!